DiamondLightSource/hyperion
# Use new graylog instance at https://graylog.diamond.ac.uk

Fixes #801
Updates Hyperion to log to the new graylog instance

Link to dodal PR (if required): https://github.com/DiamondLightSource/dodal/pull/451

### To test:
Run:
```python
from hyperion.log import LOGGER, do_default_logging_setup
do_default_logging_setup()
LOGGER.info("hello world")
```
and see the message appears in the Hyperion graylog stream: https://graylog.diamond.ac.uk/streams/66264f5519ccca6d1c9e4e03/search
